src/renderer/components/+helm-charts/helm-chart-details.tsx

```diff
@@ -118,14 +119,32 @@ class NonInjectedHelmChartDetails extends Component<HelmChartDetailsProps & Depe
     this.props.hideDetails();
   }
 
+  private renderIcon(chart: HelmChart) {
+    const icon = chart.getIcon();
+
+    if (!icon || this.imgLoadingFailed.has(chart.getId())) {
+      return (
+        <div
+          className="intro-logo"
+          dangerouslySetInnerHTML={{ __html: HelmLogoPlaceholder }}
+        />
```

Comment:
Can't we use `img` with `HelmLogoPlaceholder` as its `src` to avoid using something like `dangerouslySetInnerHTML={{ __html: HelmLogoPlaceholder }}`

Reply:
No because we currently bundle the `.svg` as the raw avg data, not as a data URL. And we have to continue bundling like that because we have to keep `<Icon>` from breaking.

I could factor out that and make it a seperate component of we want.